static/scss/partials/main.scss

```diff
@@ -334,12 +334,41 @@ table {
 .relay-address:hover,
 .relay-address:focus {
     opacity: .75;
+}
+
+.alias-on-desktop {
+    display: none;
+
+    @media #{$mq-md} {
+        display: inline;
+    }
+}
+
+.alias-on-mobile {
+    display: inline;
 
-    .relay-address--label {
-        text-decoration: underline;
+    @media #{$mq-md} {
+        display: none;
     }
 }
```

Comment:
@maxxcrawford  Yup, I switch the visibility of the mobile and desktop view here.